ci-operator/step-registry/openshift-qe/concurrent-builds/openshift-qe-concurrent-builds-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+python --version
+pushd /tmp
+python -m virtualenv ./venv_qe
+source ./venv_qe/bin/activate
+
+ES_PASSWORD=$(cat "/secret/password")
+ES_USERNAME=$(cat "/secret/username")
+
+git clone https://github.com/cloud-bulldozer/e2e-benchmarking --depth=1
+pushd e2e-benchmarking/workloads/kube-burner-ocp-wrapper
```

Comment:
this workload is not set up for this folder, should be kube-burner. The list of workloads that are curently supported using this new folder/way of running is here: https://cloud-bulldozer.github.io/kube-burner/v1.7.6/ocp/

We need to discuss if we need to move this to our svt repo because there have been talks of getting rid of the kube-burner folder